home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/util/ValidationHelper.java

```diff
@@ -0,0 +1,24 @@
+package com.softserveinc.ita.homeproject.homeservice.util;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class ValidationHelper {
+
+    /**
+     * ^                 # start-of-string
+     * (?=.*[0-9])       # a digit must occur at least once
+     * (?=.*[a-z])       # a lower case letter must occur at least once
+     * (?=.*[A-Z])       # an upper case letter must occur at least once
+     * (?=.*[@#$%^&+=])  # a special character must occur at least once
+     * (?=\S+$)          # no whitespace allowed in the entire string
+     * .{8,}             # anything, at least eight places though
+     * $                 # end-of-string
+     */
+    private static final String PASSWORD_VALIDATION_REGEX
+            = "^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=])(?=\\S+$).{8,}$";
+
+    public boolean isValidPassword(String password) {
```

Comment:
Not strict. Please, consider to use name this method as ex. “validatePasswordComplexity”, etc. 
This class can be extended in the future so we can use other validation logics. It would be great if we’ll know what exactly some method by it’s name

Reply:
Updated method name to `validatePasswordComplexity`.